.dotnet/src/Custom/Audio/AudioClient.cs

```diff
@@ -17,7 +17,8 @@ namespace OpenAI.Audio;
 [CodeGenSuppress("CreateTranslation", typeof(AudioTranslationOptions))]
 public partial class AudioClient
 {
-    private readonly string _model;
+    protected readonly string _model;
+    protected readonly Uri _endpoint;
```

Comment:
An alternate approach here would be to cache a local copy in the Azure client, since those come in through constructors, and (I think?) are only used for the request creation helpers that the Azure client will implement internal to that assembly.  If you decide to make these protected, I believe the FDG says they need to be properties and not fields and follow naming conventions of that.

Reply:
Great point; let me try doing it with new "copies." Given we're using the central protected constructor, I don't think that should pose an issue.

If there's no hassle, that'll carry the added benefit of letting the Azure version of these fields have more accurate names as appropriate, e.g. "_deploymentName" instead of "_model."